shine-examples/shine-examples.cabal

```diff
@@ -0,0 +1,32 @@
+name:               shine-examples
+version:            0.1
+build-type:         Simple
+cabal-version:      >=1.10
+license:            MIT
+
+executable shine-spaceinvaders
+    main-is:            Main.hs
+    other-modules:      Game Point
+    hs-source-dirs:     spaceinvaders
+    default-language:   Haskell2010
+    ghc-options:        -O2 -threaded -rtsopts -with-rtsopts=-N -Wall
+                        -Wno-unused-top-binds -Wno-missing-home-modules
+    ghcjs-options:      -dedupe
+    cpp-options:        -DGHCJS_BROWSER
+    build-depends:      base, containers, ghcjs-dom, keycode, random, shine
+    if !impl(ghcjs)
+        buildable:      False
+
+executable shine-mountaincar
+    main-is:            Main.hs
+    other-modules:      Game MountainCar Point
+    hs-source-dirs:     mountaincar
+    default-language:   Haskell2010
+    ghc-options:        -O2 -threaded -rtsopts -with-rtsopts=-N -Wall 
+                        -Wno-unused-top-binds -Wno-missing-signatures -Wno-missing-home-modules
```

Comment:
* Is `-O2` necessary?
* Does the `-N` actually do anything in ghcjs?
* `-Wall` is good :)
* I'm very much against disabling those warnings, except maybe missing-signatures.

edit: wait, why does this trigger -Wmissing-home-modules? Is it a ghcjs/cabal bug?

Reply:
Ok, I've updated the options.
Indeed, ghcjs seems to ignore the `other-modules` field, hence the `-Wno-missing-home-modules` option.